portafly/src/utils/index.ts

```diff
@@ -1,3 +1,4 @@
 export * from './auth'
 export * from './http'
 export * from './reducers'
+export * from './useCustomAsync'
```

Comment:
@didierofrivia @damianpm 
This file is an idea of implementation, it will not be included in the commit as it. Ideas and suggestions welcome.